.golangci.yaml

```diff
@@ -367,50 +146,12 @@ issues:
         - unconvert
         - unparam
         - unused
-
-  # The list of ids of default excludes to include or disable. By default it's empty.
-  include:
-    - EXC0002 # disable excluding of issues about comments from golint
-
-  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
+  exclude-dirs:
```

Comment:
Don't we still need `pkg/plugin/generated` to be under exclude-dirs ?

Reply:
yes

Reply:
all files in `pkg/plugin/generated` start with a comment `Code generated by ...`, so those files are excluded by default

Do you prefer to explicitly add it here?

Reply:
yeah lets keep it same as before.

Reply:
updated